modules/user/user.service.js

```diff
@@ -57,8 +70,8 @@ class UserService {
       credentials,
     };
 
-    return User.findByIdAndUpdate(id, {
-      ...checkedUser._doc,
+    return User.findByIdAndUpdate(auth.userId, {
+      ...user._doc,
```

Comment:
Do we really need to pass the old user too? Can we just put data that we need to update? Why do we need old data too?

Reply:
If we need to update some concrete field, it will concat old user with with updated fields